diff of execution of:
echo 'unset PATH; ls; /bin/ls; echo $PATH' | tr ';' '\n' | [shell]
--- expected	2024-08-27 20:21:05.415022286 +0200
+++ found	2024-08-27 20:21:05.434022500 +0200
@@ -1,8 +1,8 @@
 infile
 readonly
 
 
 ======== stderr =======
-[progname]: ls: No such file or directory
+[progname]: ls: command not found
 ======== return =======
 return value: 0
